packages/apps/spaces/app/organizations/[id]/components/Timeline/events/email/EmailPreviewModal.tsx

```diff
@@ -259,11 +252,8 @@ export const EmailPreviewModal: React.FC<EmailPreviewModalProps> = ({
             <Image
               src={'/backgrounds/organization/post-stamp.webp'}
               alt='Email'
-              width={54}
+              width={48}
               height={70}
-              style={{
```

Comment:
Is this change consulted with @steynviljoen67 ?

Reply:
We most probably covered this in a pairing session earlier, but not sure what exactly what UI change this is.

Reply:
This is all made with him while pairing.